src/app/utils/parseCommaSeparatedValues.ts

```diff
@@ -0,0 +1,6 @@
+export const parseCommaSeparatedValues = (input: string): string[] => {
+  return input
+    .split(",")
+    .map((s) => s.trim())
+    .filter(Boolean);
```

Comment:
What does this line do?

Reply:
`Boolean` converts any given value to a boolean type. It returns true for truthy values and false for falsy values. Here it's used as an argument in `.filter()`effectively filtering out all falsy values from the array.

Reply:
Examples below might help if still unclear.

```tsx
"first, , third".split(",")
    .map((s) => s.trim())
```
`['first', '', 'third']`

---

```tsx
"first, , third".split(",")
    .map((s) => s.trim())
    .filter(Boolean);
```
`['first', 'third']`